furnace/related_worker.py

```
@@ -0,0 +1,151 @@
+
+from arxiv import arxiv, SortCriterion, SortOrder
+from jinja2 import Template
+# This is a usefull tool which may help you retrieve related paper info. See the demo web page (generated_page.html)
+def simple_query_arxiv():
+    rst = []
+    query = f'all:"infrared" AND all:"small" AND all:"Target detection"'
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)